aws/acm.go

```diff
@@ -34,20 +36,47 @@ func (p *acmCertificateProvider) GetCertificates() ([]*certs.CertificateSummary,
 	return result, nil
 }
 
-func getACMCertificateSummaries(api acmiface.ACMAPI) ([]*acm.CertificateSummary, error) {
+func getACMCertificateSummaries(api acmiface.ACMAPI, filterTag string) ([]*acm.CertificateSummary, error) {
 	params := &acm.ListCertificatesInput{
 		CertificateStatuses: []*string{
 			aws.String(acm.CertificateStatusIssued),
 		},
 	}
 	acmSummaries := make([]*acm.CertificateSummary, 0)
+
 	err := api.ListCertificatesPages(params, func(page *acm.ListCertificatesOutput, lastPage bool) bool {
 		acmSummaries = append(acmSummaries, page.CertificateSummaryList...)
 		return true
 	})
+
+	if tag := strings.Split(filterTag, "="); filterTag != "=" && len(tag) == 2 {
```

Comment:
minor: I think for readability it's better to create two if's with the same return here.
The first if will check filterTag, the second will split and check len.
With such two if's, the `filterTag != "="` won't be lost somewhere in the middle of the line and would be much easier to notice.

Reply:
I don't see how `filterTag != "=" ` can help in the condition. I think `len(tag)==2` is enough, because if  `filterTag == "=" ` then it will have 3 parts after split.

Reply:
Ah https://go.dev/play/p/oGRkFVn_zu9 now I got it :D